### DISCUSSION TITLE
Multi-target TMS simulation

### DISCUSSION THREAD

**Discussion by yyx1027** [simnibs/simnibs#273] [8. Simulation workflows] [answered]
Dear SimNIBS experts,
I want to carry out TMS simulation of multiple targets, but only single simulation of different targets can be displayed during the simulation. Is there any way to observe the electric field distribution of multiple targets stimulation at the same time?

Would you kindly suggest how it could be fixed?
Thanks, regards,

**Comment by bjornsig** [comment, marked as answer]
Hi, you will have to do this in a post-processing step by adding the vectors of all the simulations. simnibs does not do it for you unfortunately. Here is an example of a post-processing step, it doesn't do vector addition but shows you how to work with the simulation results.
https://github.com/simnibs/simnibs/blob/master/simnibs/examples/simulations/TI.py

**Comment by yyx1027** [comment]
Thank you very much for your answer！

**Reply by BiomedKitty** [reply]
Hi yyx1027! I am also trying to work with multiple tms coils for a project I am doing. I read the response given to you and I looked at trying to work with the code provided but I am fairly new to coding. I was wondering if you had any luck with the code provided or could give direction on how you used/modified the code to work for you? 

Thanks!
